src/NuGet/Microsoft.CodeAnalysis.CSharp.Workspaces.nuspec

```diff
@@ -27,6 +27,8 @@ Supported Platforms:
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.xml" target="lib\portable-net45+win8" />
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" target="lib\net45" />
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.xml" target="lib\net45" />
+    <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" target="lib\netstandard1.3" />
```

Comment:
Targetting 4.5 was always a bit of a lie because we didn't have a better TFM to use.  In NetStandard we decided to be honest about our dependencies.